persistence/src/test/java/edu/unc/lib/dl/persist/services/edit/EditFilenameServiceTest.java

```diff
@@ -154,6 +151,7 @@ public void editFilenameTest() {
         service.editLabel(agent, pid, label);
 
         verify(repoObjFactory).createExclusiveRelationship(eq(binaryObj), eq(Ebucore.filename), any(Resource.class));
```

Comment:
You shouldn't need to verify this method was called twice in a row, so you should be able to get rid of this line. Also I don't think you're verify the value before it is getting overwritten by line 156 before you verify it. Have you tried just doing `verify(repoObjFactory).createExclusiveRelationship(eq(binaryObj), eq(Ebucore.filename), eq(label))` ? If that doesn't work, then you'll need another assertEquals for the label right after this capture.

You'll want to make the same changes in the other test.

Reply:
eq(label) seems to work fine. Updated